staging/src/k8s.io/apimachinery/pkg/runtime/serializer/protobuf/protobuf.go

```diff
@@ -407,6 +407,9 @@ func unmarshalToObject(typer runtime.ObjectTyper, creater runtime.ObjectCreater,
 	if err := proto.Unmarshal(data, pb); err != nil {
 		return nil, actual, err
 	}
+	if actual != nil {
```

Comment:
Hrm.  This was deliberate, but we since started to argue more that clearing gvk in memory wasn't really necessary.

I'm wondering whether anyone would be impacted.

Reply:
not being able to round trip without external fixup seemed bad